requirements.txt

```diff
@@ -1,3 +1,4 @@
 anymarkup
 lockfile
 jsonpointer
+requests
```

Comment:
@kadel @dustymabe 
Ah, yeah, that should be good, just tested it under Debian and it works. I was a tad confused for a second.